etcdutl/snapshot/v3_snapshot.go

```diff
@@ -352,38 +352,38 @@ func (s *v3Manager) modifyLatestRevision(bumpAmount uint64) error {
 	return nil
 }
 
-func (s *v3Manager) unsafeBumpRevision(tx backend.UnsafeWriter, latest revision, amount int64) revision {
+func (s *v3Manager) unsafeBumpRevision(tx backend.UnsafeWriter, latest mvcc.BucketKey, amount int64) mvcc.BucketKey {
```

Comment:
Would it make it more clear?
```suggestion
func (s *v3Manager) unsafeBumpRevisionInBucket(tx backend.UnsafeWriter, latest mvcc.BucketKey, amount int64) mvcc.BucketKey {
```